storage/postgres/migrations/20190829101500_add_platforms_status.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE platforms ADD COLUMN active boolean NOT NULL DEFAULT '0';
+ALTER TABLE platforms ADD COLUMN last_active TIMESTAMP NOT NULL DEFAULT '1970-01-01 00:00:00+00';
```

Comment:
that's strange, better use NULL is the platform was never active

Reply:
If we use NULL then in Platform struct we need to use sql.NullString instead of time.Time so rows scanning can be done properly for null values and deal with casting, which for me seems like unnecessary complication but i am okay with both solutions.